src/test/kotlin/uk/gov/justice/digital/hmpps/notificationsalertsvsip/integration/domainevents/PrisonVisitsEventsTest.kt

```diff
@@ -1,53 +1,246 @@
 package uk.gov.justice.digital.hmpps.notificationsalertsvsip.integration.domainevents
 
+import org.assertj.core.api.Assertions
```

Comment:
a lot of duplication in your test, would try and consolidate into a couple of test methods

Reply:
updated